pages/web-scraping/index.tsx

```diff
@@ -17,13 +17,22 @@ import { useVisitorData } from '@fingerprintjs/fingerprintjs-pro-react';
 import Link from 'next/link';
 import { useQueryState } from 'use-location-state/next';
 import { useQuery, UseQueryResult } from 'react-query';
-import { NextPage } from 'next';
+import { GetServerSideProps, NextPage } from 'next';
 import { FlightQuery } from '../api/web-scraping/flights';
 import { CheckResultObject } from '../../server/checkResult';
 
 // Make URL query object available as props to the page on first render
 // to read `from`, `to` params and a `disableBotDetection` param for testing and demo purposes
-export { getServerSideProps } from 'use-location-state/next';
```

Comment:
I thought this passed all query params as props to the page automatically but it's actually just an empty function that turns the page SSR and you need to pass the props manually.